vignettes/ci.Rmd

```diff
@@ -97,15 +97,12 @@ The following template can be used as a base when using renv with
 
 ```
 variables:
-  RENV_CONFIG_REPOS_OVERRIDE: "http://cran.r-project.org"
```

Comment:
If this were to stay as an example, then `https://packagemanager.posit.co/cran/latest` would be much preferred.

However, it wasn't required, since `renv` generally already embeds repo options as the defaults.